src/mixins/container.scss

```diff
@@ -0,0 +1,22 @@
+@import './mixinLayout';
+@import '../variables.scss/devicesWidth.scss';
+@import '../variables.scss/devicesPadding.scss';
+
+@mixin contentPaddingInline() {
+  padding-inline: $mobile-inline-padding;
+  max-width: 320px;
+  margin-inline: auto;
+
+  @include onTablet() {
+    max-width: 592px;
```

Comment:
I think there's no need to set those max-widths for mobile and tablet:
![image](https://github.com/fe-oct23-team3/product_catalog/assets/147749101/095f2fd6-0338-47d5-92d1-24d127fd6e0f)
![image](https://github.com/fe-oct23-team3/product_catalog/assets/147749101/628879dc-2373-41cb-b1a3-78480508a7da)
as I understood it should stretch

It only makes sense for desktop:
![image](https://github.com/fe-oct23-team3/product_catalog/assets/147749101/f3e48d8d-77a1-43e2-90ae-fcbb23d058bf)